lib/common.py

```diff
@@ -771,11 +771,12 @@ def migrate(self, target_host, sr=None):
         self.previous_host = self.host
         self.host = target_host
 
-    def snapshot(self):
+    def snapshot(self, ignore_vdis=None):
```

Comment:
Maybe `ignored_vdis`. Otherwise I have the impression that the param is a boolean.

Reply:
This is to reflect the call in xapi (the param is called `ignore_vdis` & `ignore-vdi-uuids`)